docs/content/developer/getting-started/publish.mdx

```diff
@@ -22,6 +24,14 @@ To publish your package, use the IOTA client's `publish` command. Execute this c
 iota client publish
 ```
 
+:::info Connect Wallet
+
+You can also connect your wallet to publish your package. You will see the objects directly in your wallet instead of the terminal output.
```

Comment:
```suggestion
You can connect your wallet to publish your package using its bytecode and the IOTA dApp Kit. You will see the objects directly in your wallet instead of the terminal output.
```